index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <section class="rwapper">
+      <header>
+        <nav>
+          <p class="mylogo">My Logo</p>
+          <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+        </nav>
+      </header>
+      <section class="main">
+        <main>
+          <h1>
+            I'm Emmanuel Glad to see <br />
+            you!
+          </h1>
+          <p class="txt">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hesitate to
+            contact me.
+          </p>
```

Comment:
Please consider making use of the same styles as used in Figma to make it exactly as similar to the design as possible.


## FIGMA DESIGN:-

![image](https://user-images.githubusercontent.com/94223367/234618713-8afd2912-c4c0-4f21-aeff-55bd9ce4605e.png)


## YOUR DESIGN:-

![image](https://user-images.githubusercontent.com/94223367/234618758-90408783-39e5-44da-85be-2f1dde85ad71.png)

Reply:
@ABDULALI3468  I don't understand, I have used the same properties from figma!?

Reply:
Done